frontend/src/shared/components/tasque-team-select/tasque-team-select.component.ts

```diff
@@ -0,0 +1,90 @@
+import { Component, OnInit } from '@angular/core';
+import { UserCircle } from './models';
+
+@Component({
+  selector: 'tasque-team-select',
+  templateUrl: './tasque-team-select.component.html',
+  styleUrls: ['./tasque-team-select.component.sass']
+})
+export class TasqueTeamSelectComponent implements OnInit {
+
+  public inputSearch = '';
+
+  public showPopUp = false;
+
+  public avatars: UserCircle[] = [
```

Comment:
Could you add marks on the code fragments, that should be deleted when the component gets integrated somewhere ?

Reply:
Added marks for avatars array, that should be deleted